.github/workflows/setup-workflows.yml

```diff
@@ -0,0 +1,55 @@
+name: Setup and update common workflows
+
+on:
+  schedule:
+  - cron: "0 2 * * *"
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout default branch
+        # https://github.com/marketplace/actions/checkout
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GH_WORKFLOWS_TOKEN }}
+
+      - name: Checkout common workflows repository
+        # https://github.com/marketplace/actions/checkout
+        uses: actions/checkout@v2
+        with:
+          path: ckeditor4-workflows-common
+          repository: ckeditor/ckeditor4-workflows-common
+          ref: master
+
+      - name: Setup workflows directory
+        run: |
+          mkdir -p .github/workflows
+
+      - name: Copy workflow files
+        run: |
+          rsync -a --include='*/' --include='*.yml' --exclude='*' ./ckeditor4-workflows-common/workflows/ ./.github/workflows/
+          if [[ $(git status ./.github/workflows/ --porcelain) ]]; then
```

Comment:
`git status` should be used with `./.github/workflows/` - so I updated it.

Reply:
Valid point, we somehow missed it in the past :+1: Theoretically, since it's run on clean machine on CI there shouldn't be any other changes but such defensive approach still would protect us from unexpected issue in the future :+1: 

However, as `setup-workflows` is also self updating, this change would be overwritten on first workflows update. All such changes should be done in [https://github.com/ckeditor/ckeditor4-workflows-common](https://github.com/ckeditor/ckeditor4-workflows-common) repository. Could you extract it there?

Reply:
It's already there :)  
I just update it "just in case" before the first sync.

Reply:
Ahhh, ok, sorry for the confusion :+1:  So you aligned this file with what we have on `master` in `workflows-common` which is good :+1: